addons/account/models/account_report.py

```diff
@@ -19,6 +19,13 @@
 ]
 
 DOMAIN_REGEX = re.compile(r'(-?sum)\((.*)\)')
+ACCOUNT_CODES_ENGINE_SPLIT_REGEX = re.compile(r"(?=[+-])")
+ACCOUNT_CODES_ENGINE_TERM_REGEX = re.compile(
+    r"^(?P<sign>[+-]?)"
+    r"(?P<prefix>([A-Za-z\d.]*|tag\([\w.]+\))((?=\\)|(?<=[^CD])))"
+    r"(\\\((?P<excluded_prefixes>([A-Za-z\d.]+,)*[A-Za-z\d.]*)\))?"
+    r"(?P<balance_character>[DC]?)$"
+)
```

Comment:
It's unlikely that anyone was using those constraint in a custo .... but it's not certain. So, moving them would then break that :/ We've lived with this for years ; wouldn't it be enough to only fix it in 19 ? (which is so fresh we can assume there is no custo on it yet)